bedrock/firefox/templates/firefox/firefox-20th/index.html

```diff
@@ -0,0 +1,156 @@
+  {% set page_title = 'Firefox a 20 ans - Célébrez avec nous' %}
+  {% set page_desc = 'Célébrez les 20 ans de Firefox, plongez dans notre parcours, explorez les créations de fans, rencontrez les meilleurs contributeurs et rejoignez-nous pour un futur palpitant.' %}
+  {% set main_title = 'Firefox fête ses 20 ans' %}
+  {% set main_body = 'Que vous veniez de télécharger Firefox, ou que vous soyez à nos côtés depuis le début, vous contribuez grandement à notre mission de créer une meilleure version d’Internet. Nous avons hâte de vous dévoiler la suite.' %}
+  {% set cta_title = 'Mettez du peps dans votre navigation' %}
+  {% set cta_body = 'Découvrez notre édition spéciale de fonds d’écran. Ouvrez un nouvel onglet et cliquez sur la roulette en haut à droite.' %}
+  {% set cta_default = 'Faites de Firefox votre navigateur par défaut' %}
+  {% set cta_download = 'Télécharger Firefox' %}
+  {% set fan_title = 'Les fans de Firefox ont du talent' %}
+  {% set fan_body = 'On ne se lassera jamais de votre fan art.' %}
+{% elif LANG == "de" %}
+  {% set page_title = 'Firefox wird 20 - Feiere mit uns' %}
+  {% set page_desc = 'Feiere 20 Jahre Firefox — komm mit auf unsere Reise, entdecke Werke von Fans, triff die wichtigsten Mitwirkenden und begleite uns auf dem Weg in eine spannende Zukunft.' %}
+  {% set main_title = 'Feiere 20 Jahre Firefox mit uns' %}
+  {% set main_body = 'Ob du Firefox gerade erst runtergeladen hast, oder von Anfang an dabei bist – du spielst eine wichtige Rolle dabei, das Internet zu einem besseren Ort zu machen. Wir freuen uns schon darauf, dir zu zeigen, was als Nächstes kommt.' %}
+  {% set cta_title = 'Mach deinen Browser bunter' %}
+  {% set cta_body = 'Schau dir unsere speziellen Hintergrundbilder an – öffne einen neuen Tab und klicke oben rechts auf das Zahnrad-Symbol.' %}
+  {% set cta_default = 'Firefox als Standardbrowser festlegen' %}
+  {% set cta_download = 'Firefox herunterladen' %}
+  {% set fan_title = 'Firefox-Fans haben Talent' %}
+  {% set fan_body = 'Wir können uns an eurer Fan-Art einfach nicht sattsehen.' %}
+{% else %}
+  {% set page_title = 'Firefox Turns 20 - Celebrate with Us' %}
+  {% set page_desc = 'Celebrate 20 years of Firefox — dive into our journey, explore fan creations, meet top contributors and join us as we look forward to an exciting future.' %}
+  {% set main_title = 'Celebrating 20 years of Firefox' %}
+  {% set main_body = 'Whether you just downloaded Firefox, or you’ve been with us since the beginning, you are a vital part of helping us make the internet a better place. We can’t wait to show you what’s coming next.' %}
+  {% set fan_title = 'Firefox fans are talented' %}
+  {% set fan_body = 'We’ll never get tired of your fan art.' %}
+{% endif %}
+
+{% block page_title_full %}
+  {{ page_title }}
+{% endblock %}
+{% block page_desc %}
+  {{ page_desc }}
+{% endblock %}
+{% block page_image %}
+  {{ static('img/firefox/firefox-20th/firefox-20th-social-share.png') }}
+{% endblock %}
```

Comment:
Nit: inline these to remove blank space before/after the content
```suggestion
{% block page_title_full %}{{ page_title }}{% endblock %}
{% block page_desc %}{{ page_desc }}{% endblock %}
{% block page_image %}{{ static('img/firefox/firefox-20th/firefox-20th-social-share.png') }}{% endblock %}
```